offload/test/offloading/fortran/target-map-dynamic.f90

```diff
@@ -0,0 +1,33 @@
+! Offloading test checking lowering of arrays with dynamic extents.
+! REQUIRES: flang, amdgpu
+
+! RUN: %libomptarget-compile-fortran-run-and-check-generic
+
+subroutine test_array_target_enter_data(dims)
+    integer, intent(in) :: dims(2)
+    double precision :: A(2, dims(2))
+    !$omp target enter data map(to: U)
```

Comment:
Should this be `map(to: A)` ? I don't see a variable `U`.

Reply:
It should be, thank you for the catch! I've done the same mistake a few times and seen it in a few other tests, and now I always use implicit none... :-)

Reply:
Sorry about that, fixed in https://github.com/llvm/llvm-project/commit/a67b6e163528ea547ccbcd4850803f062df66d2e.